input/ignoreWarnings.txt

```diff
@@ -4,3 +4,7 @@
 The HTML fragment 'ip-statements.xhtml' is not included anywhere in the produced implementation guide
 An HTML fragment from the set [cross-version-analysis.xhtml, cross-version-analysis-inline.xhtml] is not included anywhere in the produced implementation guide
 The HTML fragment 'globals-table.xhtml' is not included anywhere in the produced implementation guide
+
+# === These warnings are suppressed as the IG Publisher cannot validate code systems and value sets until a package dependency is available from NCTS. There is no timeline for completion of this enhancement. For more details see JIRA-48933 (https://jira.hl7.org/browse/FHIR-48933). ===
```

Comment:
I think it would be helpful to include more details about why the warnings appear - the IG Publisher cannot validate code systems and value sets against NCTS profiles because it cannot access them. 
This is implied by the current note but not explicitly stated.